src/scss/_variables.scss

```diff
@@ -1,6 +1,7 @@
 $o-colors-is-silent: true !default;
 
-$_o-colors-brand: if(global-variable-exists(o-brand), if($o-brand != null, $o-brand, 'master'), 'master');
+$_available-brands: ('master', 'internal', 'whitelabel');
+$_o-colors-brand: if((global-variable-exists(o-brand) and index($_available-brands, $o-brand)), $o-brand, 'master');
```

Comment:
Should we throw an error if the o-brand value is not in the list of available brands?